cmd/geth/log_display_dash.go

```diff
@@ -85,13 +86,19 @@ var (
 )
 
 
-func tuiDrawDash(e *eth.Ethereum) {
+func tuiDrawDash(e *eth.Ethereum, ctx *cli.Context) {
 	if currentMode == lsModeImport || currentMode == lsModeDiscover {
 		syncheightGauge.Label = ""
 	}
 	if e != nil && e.IsListening() {
 		cb := e.BlockChain().GetBlockByNumber(currentBlockNumber)
-		syncheightGauge.BorderLabel = fmt.Sprintf("%s | local_head ◼ n=%d ⬡=%s txs=%d time=%v ago", currentMode, currentBlockNumber, cb.Hash().Hex()[:10] + "…", cb.Transactions().Len(), time.Since(time.Unix(cb.Time().Int64(), 0)).Round(time.Second))
+		cid := e.ChainConfig().GetChainID()
+		cnet := e.NetVersion()
+		// Is there any other way retrieve chain name(mainnet or morden)?
```

Comment:
Could use `cacheChainIdentity`, since it will have been set during node configuration.